pkg/kubevip/config_manager.go

```diff
@@ -203,16 +202,16 @@ func isValidInterface(iface string) error {
 	if err != nil {
 		return fmt.Errorf("get %s failed, error: %w", iface, err)
 	}
+	attrs := l.Attrs()
 
-	if l.Attrs().Flags&net.FlagLoopback == net.FlagLoopback {
-		return nil
-	}
-
-	if l.Attrs().Flags&net.FlagPointToPoint == net.FlagPointToPoint {
-		return nil
-	}
-
-	if l.Attrs().OperState != netlink.OperUp {
+	// Some interfaces (included but not limited to lo and point-to-point
+	//	interfaces) do not provide a operational status but are safe to use.
+	// From kernek.org: "Interface is in unknown state, neither driver nor
+	// userspace has set operational state. Interface must be considered for user
+	// data as setting operational state has not been implemented in every driver."
+	if attrs.OperState == netlink.OperUnknown {
+		log.Warningf("the status of the interface %s is unknown. Ensure your interface is ready to accept traffic, if so you can safely ignore this message")
```

Comment:
Linting error here :-(

Reply:
My golangci-lint kept crashing (turnouts it doesn't like being installed via Nix) so I didn't notice. Sorry